docs/whatsnew/3.0.rst

```diff
@@ -38,3 +38,52 @@ Full change log
 
 To see a detailed list of all changes in version v3.0, including changes in
 API, please see the :ref:`changelog`.
+
+
+Renamed/removed functionality
+=============================
+
+pytest plugins
+**************
+
+The following pytest plugins were previously provided as part of the Astropy
+core package but have now been moved to separate packages:
+
+* `pytest-remotedata <https://github.com/astropy/pytest-remotedata>`__
+* `pytest-doctestplus <https://github.com/astropy/pytest-doctestplus>`__
+* `pytest-openfiles <https://github.com/astropy/pytest-openfiles>`__
+
+Affiliated packages that make use of one or more of these plugins may install
+them individually using ``pip``, or they may install the Astropy testing
+package ``pytest-astropy``, which includes all of these plugins as
+dependencies. Packages that use Astropy's test runner will also need to install
+``pytest-astropy``. See :ref:`testing-dependencies` for more details.
+
+The ``config`` and ``display`` plugins are still provided as part of the
+Astropy core package, but they have moved from ``astropy.tests.pytest_plugins``
+to ``astropy.tests.plugins.config`` and ``astropy.tests.plugins.display``,
+respectively.
+
+Affiliated packages that currently access plugin decorators and other variables
+from ``astropy.tests.pytest_plugins`` will need to make updates, although
+backwards compatibility will be maintained in the meantime.
+
+Below is an outline of the required changes:
+
+* It is no longer necessary to import ``remote_data`` from ``astropy.tests.helper``
```

Comment:
This is confusing for affiliated packages. Write explicitly what one should do to ensure one can mark `@remote_data` and get no errors upon execution (from `flake8` if nothing else)

Reply:
Thanks for the feedback. Hopefully this is clearer now.

Reply:
Yes, now it is very clear! Thanks.